reader.go

```diff
@@ -135,7 +135,7 @@ func (r *Reader) ReadMIMEHeader() (MIMEHeader, error) {
 			// Most headers aren't multi-valued.
 			// Set the capacity on pairs[0] to 1, so any future append
 			// won't extend the slice
-			pairs = make([]KVPair, 1, 1)
+			pairs = make([]KVPair, 1)
```

Comment:
Could you let me know why you removed capacity? The comment above has an explanation for it. Thus if the reason for removing is the linter, then it would be better either ignore the lint check for the line or add an explanation for the change.

Reply:
It was for the linter, missed the comment, reverted.

Reply:
done